addons/point_of_sale/views/pos_order_report_view.xml

```diff
@@ -10,6 +10,7 @@
                     <field name="order_id" type="measure"/>
                     <field name="product_qty" type="measure"/>
                     <field name="price_total" type="measure"/>
+                    <field name="margin"/>
```

Comment:
maybe you want it of type="measure", like in the graph view

Reply:
After some testing, this is not needed. I don't know why I've put this anymore. The one in the graph view is not needed too